test/unit/bin-tessel-2.js

```diff
@@ -239,9 +238,12 @@ exports['Tessel (cli: wifi)'] = {
   setUp: function(done) {
     this.sandbox = sinon.sandbox.create();
     this.warn = this.sandbox.stub(logs, 'warn');
+    this.info = this.sandbox.stub(logs, 'info');
     this.printAvailableNetworks = this.sandbox.stub(controller, 'printAvailableNetworks').returns(Promise.resolve());
     this.connectToNetwork = this.sandbox.stub(controller, 'connectToNetwork').returns(Promise.resolve());
-    this.processExit = this.sandbox.stub(process, 'exit');
```

Comment:
For some reason, `process.exit` wasn't being stubbed over properly here. The actual `process.exit` would get called and break the tests. I opted to use `closeSuccessfulCommand` insteaad.